res/style.css

```diff
@@ -621,29 +689,58 @@ body {
 }
 
 .tabBarTab {
+  font-size: 12px;
   width: 8em;
   text-align: center;
   cursor: default;
   -moz-user-select: none;
   text-align: center;
-  padding: 4px;
-  border: solid #D6D6D6;
-  border-width: 0 1px;
-  margin: 0 -0.5px;
+  padding: 6px 4px;
   white-space: nowrap;
   background-clip: padding-box;
+  transition: background-color 200ms, border-color 200ms;
+  transition-timing-function: var(--animation-timing);
+  position: relative;
+  border: solid transparent;
+  border-width: 0 1px 0 1px;
 }
-
 .tabBarTab.selected {
-  background: white;
+  background: #fff;
+  color: var(--blue-50);
```

Comment:
A11y: the contrast is not good enough for small text on a white background.

Reply:
same for the top bar, the transform bar, the selected tab.

Reply:
Thanks for checking this. The photon colors state that "--blue-50" is "AA Large" which is for large text only. I popped it down to "--blue-60" which is then "AA" for small text.